sixtyfps_runtime/rendering_backends/qt/qt_window.rs

```diff
@@ -176,6 +176,13 @@ cpp! {{
         }
     };
 
+    class SixtyFPSPublicGraphicsBlurEffect : public QGraphicsBlurEffect {
+    public:
+        // Make public what's protected
+        void draw_public(QPainter *p) {
```

Comment:
`using QGraphicsBlurEffect::draw;`

Reply:
Makes perfect sense :). Thanks, I'm too old fashioned sometimes.

Reply:
That's good old C++98